test/unit/error_events.test.js

```diff
@@ -96,6 +88,33 @@ test('Error events', (t) => {
       })
     })
 
+    t.test('should include spanId agent attribute', (t) => {
```

Comment:
But while I was in here... figured I'd add a check for the spanId attribute.

Reply:
so extra! :D